### PR TITLE
Add omg.lol

### PR DESCRIPTION
**Pull request by coreystone**
Home page: [omg.lol](https://home.omg.lol/)
Similarweb.com ranking: [#157,093](https://www.similarweb.com/website/omg.lol/#overview)

Credit belongs to @newbold for the .svg file.

### CODE REVIEW

File: img/o/omg.lol.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...t:10;stroke-dasharray:none;stroke-opacity:1"/></g><g transform="matrix(1.88679 0 0 -1.88679 303.066 176.636)"><path d="M0 0a6.25 6.25 0 1 0-12.5 0A6.25 6.25 0 0 0 0 0" style="fill:#461036;fill-opacity:1;fill-rule:nonzero;stroke:none"/><path d="M0 0a6.25 6.25 0 1 0-12.5 0A6.25 6.25 0 0 0 0 0Z" style="fill:none;stroke:#461036;stroke-width:6;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:10;stroke-dasharray:none;stroke-opacity:1"/></g><g transform="matrix(1.88679 0 0 -1.88679 208.749 223.818)"><path d="M0 0c10.004-16.325 33.722-16.325 43.726 0" style="fill:none;stroke:#461036;stroke-width:10;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;stroke-dasharray:none;stroke-opacity:1"/></g><g transform="matrix(1.88679 0 0 -1.88679 438.679 212.013)"><path d="M0 0c0 17.259-13.991 31.25-31.25 31.25S-62.5 17.259-62.5 0s13.991-31.25 31.25-31.25S0-17.259 0 0" style="fill:#e34198;fill-opacity:1;fill-rule:nonzero;stroke:none"/></g><g transform="matrix(1.88679 0 0 -1.88679 179.245 212.013)"><path d="M0 0c0 17.259-13.991 31.25-31.25 31.25S-62.5 17.259-62.5 0s13.991-31.25 31.25-31.25S0-17.259 0 0" style="fill:#e34198;fill-opacity:1;fill-rule:nonzero;stroke:none"/></g></svg>
```
Comment (Carlgo11) [inline img/o/omg.lol.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 50 500 400"><g id="a"><path fill="#ff69ad" d="M153.06 50.03a149.56 149.56 0 0 0-109.13 43.9c-58.58 58.58-58.58 153.56 0 212.14l100 100A149.53 149.53 0 0 0 250 450V88.2a149.47 149.47 0 0 0-96.94-38.17Z"/><path fill="#461036" d="M209.1 214.39a9.43 9.43 0 0 0-8.4 14.36c11.18 18.23 30.24 27.35 49.3 27.35v-18.36c-12.75 0-25.5-6.28-33.2-18.85a9.44 9.44 0 0 0-7.7-4.5z"/><path fill="#e34198" d="M179.25 212.01a58.97 58.97 0 1 0-117.93 0 58.97 58.97 0 0 0 117.93 0"/><circle cx="208.73" cy="176.64" r="17.45" fill="#461036"/></g><use href="#a" transform="matrix(-1 0 0 1 500 0)"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
As @newbold doesn't believe in international law, please exclude the EEA/EU/UK regions using a regions array.

File: entries/o/omg.lol.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "omg.lol": {
+    "domain": "omg.lol",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://home.omg.lol/info/two-factor-authentication",
+    "categories": [
+      "domains"
+    ]
```
Comment (Carlgo11) [inline entries/o/omg.lol.json]:
```suggestion
    ],
    "regions": [
    "-at",
    "-be",
    "-bg",
    "-ch",
    "-cy",
    "-cz",
    "-de",
    "-dk",
    "-ee",
    "-es",
    "-fi",
    "-fr",
    "-gb",
    "-gr",
    "-hr",
    "-hu",
    "-ie",
    "-is",
    "-it",
    "-li",
    "-lt",
    "-lu",
    "-lv",
    "-mt",
    "-nl",
    "-no",
    "-pl",
    "-pt",
    "-ro",
    "-se",
    "-si",
    "-sk"
  ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by newbold** [comment]
> As @newbold doesn't believe in international law, please exclude the EEA/EU/UK regions using a regions array.

I’d be happy to dig into this if you’re willing to clarify what the specific issue is. Thanks!

**Comment by Carlgo11** [comment]
@newbold  I read the legal page and the part about GDPR was just plain wrong. As long as EU/EEA/UK residents are allowed to use the site, GDPR applies. It doesn’t matter if the party is a business, nonprofit or person. Origin of the site also doesn’t matter.
You need to rewrite that part and specify a DPA.

**Review by Carlgo11 — APPROVED** [review]

**Comment by newbold** [comment]
> @newbold I read the legal page and the part about GDPR was just plain wrong. As long as EU/EEA/UK residents are allowed to use the site, GDPR applies. It doesn’t matter if the party is a business, nonprofit or person. Origin of the site also doesn’t matter.
> You need to rewrite that part and specify a DPA.

Thanks for the info! That content was written by an attorney, so I’ll get back with them to have it corrected. I’ll also work on providing a DPA. Thanks again!

**Comment by Carlgo11** [comment]
That's great news. Feel free to post an update when your new privacy policy is posted and I'll remove the exclusion.

Small correction: __DPA__ was a typo and should really be __DPO__ as in _Data Protection Officer_ which is a point of contact for consumers and authorities. Although a DPA might also be useful in your case as you're sending/storing data in a ["third country"](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation#Third_countries).

**Pull request merged**